x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -194,7 +196,9 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
   );
 
   const renderOperatorInput = (isFirst: boolean): JSX.Element => {
-    const operatorOptions = onlyShowListOperators
+    const operatorOptions = operatorsList
+      ? operatorsList
+      : onlyShowListOperators
```

Comment:
Does `operatorOptions` store either and array or a boolean value? It looks like it does here. Is it intentional? 🤔

Reply:
No, always an operators array:

```
const operatorOptions =  operatorsList
      ? operatorsList
      : onlyShowListOperators
      ? EXCEPTION_OPERATORS_ONLY_LISTS
      : getOperatorOptions(
          entry,
          listType,
          entry.field != null && entry.field.type === 'boolean',
          isFirst && allowLargeValueLists
        );
``` 
What means: `operatorsList` || `EXCEPTION_OPERATORS_ONLY_LISTS` || `getOperatorOptions(
          entry,
          listType,
          entry.field != null && entry.field.type === 'boolean',
          isFirst && allowLargeValueLists
        )`